docs/src/modules/components/AppDrawer.js

```diff
@@ -5,12 +5,12 @@ import { useSelector } from 'react-redux';
 import { withStyles } from '@material-ui/core/styles';
 import List from '@material-ui/core/List';
 import Drawer from '@material-ui/core/Drawer';
-import Box from '@material-ui/core/Box';
 import SwipeableDrawer from '@material-ui/core/SwipeableDrawer';
 import Divider from '@material-ui/core/Divider';
 import Hidden from '@material-ui/core/Hidden';
-import AppDrawerNavItem from 'docs/src/modules/components/AppDrawerNavItem';
+import Box from '@material-ui/core/Box';
 import DiamondSponsors from 'docs/src/modules/components/DiamondSponsors';
+import AppDrawerNavItem from 'docs/src/modules/components/AppDrawerNavItem';
```

Comment:
nit: keep the original import order for git blame

Reply:
Agree, code was removed and added in two different commits, make it harder to identify how to minimize the visual diff.